modules/gke-service-account/README.md

```diff
@@ -6,7 +6,10 @@ https://cloud.google.com/kubernetes-engine/docs/tutorials/authenticating-to-clou
 
 ## How do you use this module?
 
-* See the [root README](/README.md) for instructions on using Terraform modules.
-* See the [examples](/examples) folder for example usage.
-* See [variables.tf](./variables.tf) for all the variables you can set on this module.
-* See [outputs.tf](./outputs.tf) for all the variables that are outputed by this module.
+* See the [root README](https://github.com/gruntwork-io/terraform-google-gke/blob/master/README.md) for instructions on
+using Terraform modules.
+* See the [examples](https://github.com/gruntwork-io/terraform-google-gke/tree/master/examples) folder for example usage.
+* See [variables.tf](https://github.com/gruntwork-io/terraform-google-gke/blob/master//variables.tf) for all the
+variables you can set on this module.
+* See [outputs.tf](https://github.com/gruntwork-io/terraform-google-gke/blob/master//outputs.tf) for all the variables
```

Comment:
I think this is missing the module in the path?

Reply:
🤦‍♀ not sure what happened on these ones